dev-libs/mpc/mpc-1.1.0.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Arithmetics of complex numbers with arbitrarily high precision"
+DESCRIPTION="GNU Mpc is a C library for the arithmetic of complex numbers \
+with arbitrarily high precision and correct rounding of the result. It \
+extends the principles of the IEEE-754 standard for fixed precision real \
+floating point numbers to complex numbers, providing well-defined semantics \
+for every operation. At the same time, speed of operation at high precision \
+is a major design goal."
+HOMEPAGE="http://www.multiprecision.org"
+COPYRIGHT="2006-2017 Andreas Enge, Mickaël Gastineau, Philippe Théveny and \
+Paul Zimmermann"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://ftp.gnu.org/gnu/mpc/mpc-$portVersion.tar.gz"
+CHECKSUM_SHA256="6985c538143c1208dcb1ac42cedad6ff52e267b47e5f970183a3e75125b43c2e"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=3.1.0
+libVersionCompat="$libVersion compat >= 1"
```

Comment:
```diff
-libVersionCompat="$libVersion compat >= 1"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"

Reply:
I think there is some historical reason for this non-standard compat.
(@korli is Haiku's main gcc specialist and I think he can explain why we need to keep the `>= 1`.)

Reply:
It's just a typo @fbrosson 

https://github.com/haikuports/haikuports/blob/7cfe089a5f33297b9d3e361227f60ab12775ee0a/dev-libs/mpc/mpc-1.0.3.recipe#L21